.github/workflows/e2e_other_tests.yml

```diff
@@ -62,11 +62,6 @@ jobs:
         run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_e2e_tests --prod_env --skip-install --skip-build --suite="adminPage"
         env: 
           VIDEO_RECORDING_IS_ENABLED: 0
-      - name: Run e2e Release coordinator Page Test
```

Comment:
@brianrodri Any update on this? I'm also unsure as to why we're removing this test

Reply:
Ah that was described in [this comment](https://github.com/oppia/oppia/pull/13021#issuecomment-862064848), here's a quote:
> @DubeySandeep please note that the Release Coordinator Page e2e tests have been deleted because there aren't any jobs for it to test, and because testing jobs was the only thing it did.

Reply:
Thanks for the link, @brianrodri